lib/presentation/widget/initialize_get_it/initialize_viewmodel.dart

```diff
@@ -73,20 +76,36 @@ class InitializeViewModel extends BaseViewModel {
   ThunkAction<AppState> getCredentialAction() {
     return (Store<AppState> store) async {
       await _getCredentialInteractor.execute().then((result) => result.fold(
-          (left) => store.dispatch(getCredentialFailureAction(left)),
-          (right) => store.dispatch(getCredentialSuccessAction((right)))));
+        (left) => store.dispatch(authenticationFailureAction()),
+        (right) => store.dispatch(getCredentialSuccessAction((right)))));
     };
   }
 
   ThunkAction<AppState> getCredentialSuccessAction(CredentialViewState success) {
     return (Store<AppState> store) async {
       _dynamicUrlInterceptors.changeBaseUrl(success.baseUrl.origin);
       _retryInterceptors.setPermanentToken(success.token);
+      store.dispatch(getAuthorizedUserAction());
+    };
+  }
+
+  ThunkAction<AppState> getAuthorizedUserAction() {
+    return (Store<AppState> store) async {
+      await _getAuthorizedInteractor.execute().then((result) => result.fold(
+        (failure) => store.dispatch(authenticationFailureAction()),
```

Comment:
what happen when we can not get authorized user? navigate to what? ...

Reply:
``authenticationFailureAction()`` will navigate to login